projects/admin/src/app/circulation/patron/loan/fixed-date-form/fixed-date-form.component.ts

```diff
@@ -89,19 +94,20 @@ export class FixedDateFormComponent implements OnInit {
         (data: any) => {
           const library = new Library(data.metadata);
           this.bsConfig.daysDisabled = library.closedDays;
-          // TODO :: Try to manage the exception dates
-          //   The problem is the exception dates generation is not perfect (repeatability is not managed) and take a long time.
-          //   If user has already clicked on the input, the datePicker widget is used without this configuration (it's pity)
-          //   --> but if user choose an exception closed date, then the backend will compute the best opening day and return this value ;
-          //       if asked end_date is different from API response, the `LoanComponent` will display a toastr message to inform the staff
-          //       member.
-          // this.bsConfig.datesDisabled(library.exceptionClosedDates);
         }
       );
+      this._subscription.add(this._userService.user.currentLibraryClosedDates$.subscribe(
```

Comment:
As it is an http call subscription is not required.

Reply:
`UserService.user.currentLibraryClosedDates$` is a BehaviorSubject, this is not an http call. So I think subscription & unregister are required.

This subject emit the closed date for the current used library. When user switch for another library, an http call is done to know the closed date for a complete year (fast quick api) and the setter function `set currentLibraryClosedDates(...)` emit these closed date to subscribers.